README.md

```diff
@@ -18,72 +18,45 @@
 ```
 
 ## About
-
-Counterfit is a command-line tool and generic automation layer for assessing the security of machine learning systems.
+Counterfit is a generic automation layer for assessing the security of machine learning systems. It brings several existing adversarial frameworks under one tool, or allows users to create their own. 
 
 ### Requirements
-
-- Python 3.7 or 3.8
+- Ubuntu 18.04+
+- Python 3.8
+- Windows is supported by Counterfit, but not necessarily officially supported by each individual framework. 
 - On Windows the [Visual C++ 2019 redistributable](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads) is required
 
-## Getting Started
-
-Choose one of these methods to get started quickly:
-
-- [Option 1: Deploy via Azure Shell](#Option-1-Deploy-now-on-Azure-via-Azure-Shell)
-- [Option 2: Setup an Anaconda Python environment and install locally](#Option-2-Setup-an-Anaconda-Python-environment-and-install-locally)
-
-For more information including alternative installation instructions, please visit our [wiki](https://github.com/Azure/counterfit/wiki).
-
-### Option 1: Deploy via Azure Shell
-
-To run Counterfit from your browser
-
-1. Click the button below to initiate small resource deployment to your Azure account.
-
-[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fcounterfit%2Fmaster%2Finfrastructure%2Fazuredeploy.json)
-
-2. In the configuration blade, specify your subscription and resource group.
-3. In your [Azure Shell](https://shell.azure.com), type the following, replacing `RESOURCE_GROUP` with the name of the resource group selected in the previous step.
-
-```
-az container exec --resource-group RESOURCE_GROUP --name counterfit --exec-command '/bin/bash'
-```
+## Quick Start
 
-4. Within the container, launch Counterfit.
-
-```
-python counterfit.py
-```
-
-### Option 2: Setup an Anaconda Python environment and install locally
-
-1. Install [Anaconda Python](https://www.anaconda.com/products/individual) and [git](https://git-scm.com/downloads).
-2. Clone this repository.
-
-```
-git clone https://github.com/Azure/counterfit.git
+### Installation with Python virtual environment
+```bash
+sudo apt install python3.8 python3.8-venv
+python3 -m venv counterfit
+git clone --depth 1 -b develop/1.1 https://github.com/Azure/counterfit.git
+cd counterfit
+pip install .[dev]
 ```
 
-3. Open an Anaconda shell and create a virtual environment and dependencies.
+### Installation with Conda
 
-```
+```bash
+conda create --yes -n counterfir python=3.8.0
+conda activate counterfir
+git clone --depth 1 -b develop/1.1 https://github.com/Azure/counterfit.git
 cd counterfit
-conda create --yes -n counterfit python=3.8.8
-conda activate counterfit
-pip install -r requirements.txt
-export PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python
+conda install --file requirements.txt
```

Comment:
Some of the packages are not found in default conda channel so it would be nice if we install via PIP after activating conda environment, `pip install -r requirements.txt`
![image](https://user-images.githubusercontent.com/43563047/199292285-e979c168-657c-4cb2-bf9c-6f21caf6c7da.png).
I was able to install all required packages from `requirements.txt` via `pip`.

Reply:
@rdheekonda, latest commits should've fixed this 👍